src/interface/widget.c

```diff
@@ -294,6 +294,8 @@ static void widget_tab_draw(rct_drawpixelinfo *dpi, rct_window *w, int widgetInd
 	// Resolve the absolute ltrb
 	int l = w->x + widget->left;
 	int t = w->y + widget->top;
+	int r = w->x + widget->right;
+	int b = w->y + widget->bottom;
 
```

Comment:
Well, three of these notes are all in the same place, but I tried removing those two lines and it gave me errors.